choreography/src/delivery/Dockerfile

```diff
@@ -28,14 +28,14 @@ ENTRYPOINT ["dotnet", "test", "--logger:trx"]
 
 FROM build AS publish
 
-MAINTAINER Fernando Antivero (https://github.com/ferantivero)
+LABEL Maintainer="Fernando Antivero (https://github.com/ferantivero)"
```

Comment:
I think it is time to change this to `https://github.com/mspnp`

Reply:
Roger that, updated